src/scripts.js

```diff
@@ -93,26 +107,75 @@ function parseCardFromData(data) {
           />
         <div class="bottom-card">
           <header class="card-header">
-            <h4>${destination.destination}</h4>
+            <h1>${destination.destination}</h1>
           </header>
-          <p class="date">Trip Date: ${trip.date}</p>
-          <p class="duration">Trip Duration: ${trip.duration}</p>
-          <p class="num-travelers">Number of Travelers: ${trip.travelers}</p>
+          <p class="date">Date: ${trip.date}</p>
+          <p class="duration">Duration: ${trip.duration}</p>
+          <p class="num-travelers">Traveler(s): ${trip.travelers}</p>
+          <p class="status">Status: ${trip.status}</p>
         </div>
       </article>
     `
     return acc
   }, ``)
 }
 
-
-function getTotalCostFromPastYear() {
-  const travelerTripsFromPastYear = tripsRepository.getTravelerTripsFromPastYear(displayedTravelersId, "2022/06/11");
-  const totalCostOfTrips = travelerTripsFromPastYear.reduce((totalCost, trip) => {
+function getTotalTripCost(tripData) {
+  const totalTripCost = tripData.reduce((totalCost, trip) => {
     const tripDestination = destinationsRepository.getDestinationById(trip.destinationID);
     totalCost += (tripDestination.estimatedLodgingCostPerDay * trip.duration) + (tripDestination.estimatedFlightCostPerPerson * trip.travelers);
     return totalCost;
   }, 0)
-  const fee = totalCostOfTrips * .10;
-  return (totalCostOfTrips + fee).toFixed(2);
+  const fee = totalTripCost * .10;
+  return (totalTripCost + fee).toFixed(2);
+}
+
+function checkFormInputs(e) {
+  const formattedDate = dayjs(tripDateInput.value).format('YYYY/MM/DD')
+  const isValidDate = dayjs(formattedDate).isBefore(currentDate);
+
+  if (!isValidDate &&
+    tripDurationInput.value &&
+    tripNumTravelersInput.value &&
+    tripDestinationSelection.value) {
```

Comment:
This is such a wonderful stringing of conditional logic to make sure the user's input is valid